packages/tensorflow-ocaml/tensorflow-ocaml.0.0.1/opam

```diff
@@ -0,0 +1,17 @@
+opam-version: "1.2"
+maintainer: "Laurent Mazare <lmazare@gmail.com>"
+homepage: "https://github.com/LaurentMazare/tensorflow-ocaml"
+bug-reports: "https://github.com/LaurentMazare/tensorflow-ocaml/issues"
+dev-repo: "git+https://github.com/LaurentMazare/tensorflow-ocaml.git"
+build: [make "tensorflow.lib"]
+authors: [ "Laurent Mazare" "Nicolas Oury" ]
+
+depends: [
+  "core_kernel"
+  "ctypes" {>= "0.5"}
+  "ctypes-foreign"
+  "ocamlfind"
+  "ocamlbuild"
```

Comment:
`ocamlfind` and `ocamlbuild` can be marked as `{build}` dependencies in the future. This helps reduce the recompilation burden on users when those packages are updated.